scripts/functions.sh

```diff
@@ -408,4 +408,19 @@ function postgres_ready_status() {
   >&2 echo "Postgres is unavailable - sleeping"
   sleep 1
 done
-}
\ No newline at end of file
+}
+
+function create_gwc_tile_tables(){
+  HOST="$1"
+  PORT="$2"
+  USER="$3"
+  DB="$4"
+  POSTGRES_SCHEMA="$5"
+  if [ ${POSTGRES_SCHEMA} != 'public' ]; then
+   psql -d "$DB" -p "$PORT" -U "$USER" -h "$HOST" -c "CREATE SCHEMA IF NOT EXISTS ${POSTGRES_SCHEMA}"
+   psql -d "$DB" -p "$PORT" -U "$USER" -h "$HOST" -c "CREATE TABLE IF NOT EXISTS ${POSTGRES_SCHEMA}.tileset(key character varying(320) COLLATE pg_catalog.\"default\" NOT NULL,layer_name character varying(128) COLLATE pg_catalog.\"default\",gridset_id character varying(32) COLLATE pg_catalog.\"default\",blob_format character varying(64) COLLATE pg_catalog.\"default\",parameters_id character varying(41) COLLATE pg_catalog.\"default\",bytes numeric(21,0) NOT NULL DEFAULT 0,CONSTRAINT tileset_pkey PRIMARY KEY (key))"
```

Comment:
COLLATE pg_catalog.\"default\" is not necessary here, I think it defaults to here